dotnet/samples/Concepts/ChatCompletion/Ollama_ChatCompletionStreaming.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Ollama;
+
+namespace ChatCompletion;
+
+/// <summary>
+/// These examples demonstrate the ways different content types are streamed by Ollama via the chat completion service.
+/// </summary>
+public class Ollama_ChatCompletionStreaming(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// This example demonstrates chat completion streaming using Ollama.
+    /// </summary>
+    [Fact]
+    public Task StreamChatAsync()
+    {
+        Assert.NotNull(TestConfiguration.Ollama.ModelId);
+
+        Console.WriteLine("======== Ollama - Chat Completion Streaming ========");
+
+        var chatService = new OllamaChatCompletionService(
+            endpoint: new Uri(TestConfiguration.Ollama.Endpoint),
```

Comment:
Elsewhere we are using strings for endpoints. Why are we changing the pattern for this connector?

Reply:
Mark, currently only OpenAI connectors use `string` for endpoint information and in the V2 I kept it just for back compatibility, moving forward in the connectors I'm using Uri as this type already validates the string is a valid Uri.

[HuggingFace = Uri](https://github.com/microsoft/semantic-kernel/blob/6c0dc65d246ab30831b1f6fdbd0fcdaa08d75096/dotnet/src/Connectors/Connectors.HuggingFace/HuggingFaceKernelBuilderExtensions.cs#L34C9-L34C20)

[Mistral AI = Uri](https://github.com/microsoft/semantic-kernel/blob/6c0dc65d246ab30831b1f6fdbd0fcdaa08d75096/dotnet/src/Connectors/Connectors.MistralAI/MistralAIServiceCollectionExtensions.cs#L30)

[Azure AI Inference = Uri](https://github.com/microsoft/semantic-kernel/blob/7a9a441f01f98e0e31bec344527152d6bc024fea/dotnet/src/Connectors/Connectors.AzureAIInference/Extensions/AzureAIInferenceServiceCollectionExtensions.cs#L33)

[Anthropic = Uri](https://github.com/microsoft/semantic-kernel/blob/0a4df864d861901b63a1191c8d5421682b5cbb6e/dotnet/src/Connectors/Connectors.Anthropic/Extensions/AnthropicServiceCollectionExtensions.cs#L61)